operators/bosh-operator/DirectorService.js

```diff
@@ -219,14 +220,23 @@ class DirectorService extends BaseDirectorService {
     };
     return this
       .initialize(operation)
+      .catch(err => {
+        logger.error(`Error occurred while acquiring network index for create
```

Comment:
Do you want to push this error statement to LN 226 & add a logger.warn at LN229 with warn saying error occurred which is potentially BOSH being unavailable and deployment will continue to remain stagerred?

Reply:
done